### DISCUSSION TITLE
Is there a way to use clerk sdk to authenticate the user (using email and password)

### DISCUSSION THREAD

**Discussion by e-ramadani** [clerk/javascript#3528] [Support request - DO NOT OPEN HERE]
I have a scenario where I need to generate the session jwt token using the user's credentials (email and password). However, I don't want to explicitly send a request to the clerk sign in api. Is there a way to use the clerk sdk to handle this?

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
